airflow/operators/sensors.py

```diff
@@ -207,6 +207,50 @@ def poke(self, context):
         return count
 
 
+class NamedHivePartitionSensor(BaseSensorOperator):
+    """
+    Waits for a partition to show up in Hive. Only accepts plain partition
+    names (eg dt=xxx) for the partition argument and no filters.
+
+    :param table: The name of the table to wait for, supports the dot
+        notation (my_database.my_table)
+    :type table: string
+    :param partition: The partition clause to wait for. This is passed as
+        is to the Metastore Thrift client "get_partitions_by_name" method.
+    :type partition: string
+    :param metastore_conn_id: reference to the metastore thrift service
+        connection id
+    :type metastore_conn_id: str
+    """
+    def __init__(
```

Comment:
Probably should haven't referred to `'db.table/ds=1'` as the partition name. You're right, in the API, only the `ds=1` part is referred to as the partition name. What you'd have to do is split the identifier `'db.table/ds=1'` based on the `.` to get the DB name, and then split by slash appropriately to get the partition name.